### ISSUE TITLE
[bug] Copy and paste not working for system clipboard-supporting browsers

### ISSUE THREAD

**Issue by toyxyz** [zero01101/openOutpaint#124]
As of now, to add the image I want to the canvas, I need to go through the process of Stamp image > Upload Image. Conversely, to export the image I worked on on the canvas, I have to download it by clicking Save canvas. I don't know if it's technically possible, but if possible, it would be nice to have a function to paste and copy image layers using ctrl + c / ctrl + v like in Photoshop. PaintHua supports a similar feature. https://github.com/BlinkDL/Hua

https://user-images.githubusercontent.com/8006000/209550170-33358154-e599-46f7-8b24-441ea07d358a.mp4

**Comment by seijihariki** [comment]
If you could test the fix deployed to the `testing` branch (go to `webui/extensions/<webui-openOutpaint-ext>/app` and run `git checkout origin/testing`)

**Comment by toyxyz** [comment]
> If you could test the fix deployed to the `testing` branch (go to `webui/extensions/<webui-openOutpaint-ext>/app` and run `git checkout origin/testing`)
> 
> @toyxyz

I tested it by changing the branch. The Devtools log is:

![image](https://user-images.githubusercontent.com/8006000/209559597-3e99c2bb-1570-49d3-aad3-771fe9df10c1.png)

![image](https://user-images.githubusercontent.com/8006000/209559541-58a3e3a1-2327-4be5-ba15-37b901188120.png)

**Comment by seijihariki** [comment]
Can you clean site data and reload? `test` variable shouldn't exist anymore in line 599.

**Comment by toyxyz** [comment]
> Can you clean site data and reload? `test` variable shouldn't exist anymore in line 599.

I cleared the site data and tested again. Here's the Devtools log:

![image](https://user-images.githubusercontent.com/8006000/209563286-2f89d5fb-2fc8-4f41-94aa-57960c19c7ff.png)

**Comment by seijihariki** [comment]
Are you in commit 100829298ced8dab2aa1020e80866b16856e8d5d?

**Comment by zero01101** [comment]
![image](https://user-images.githubusercontent.com/1649724/209563528-99bacb99-c0cc-4a08-b5d4-1f149c1b61b0.png)
here's what line 599 should be on the current testing branch; did you refresh the page without cache? (ctrl+f5)

**Comment by toyxyz** [comment]
> ![image](https://user-images.githubusercontent.com/1649724/209563528-99bacb99-c0cc-4a08-b5d4-1f149c1b61b0.png) here's what line 599 should be on the current testing branch; did you refresh the page without cache? (ctrl+f5)

Oh, I checked, and when I typed `git checkout origin/testing`, it was 1008292, but when I restarted webui, it was returning to main. When I changed the branch and tested it without exiting the webui, ctrl+v worked well. thanks!

![image](https://user-images.githubusercontent.com/8006000/209565426-b308043a-74c9-4849-a230-69385663bf6d.png)
![image](https://user-images.githubusercontent.com/8006000/209565625-92dfbfd2-68f3-4dc6-b7a1-904c0701be92.png)
